### PR TITLE
Print out the VOL connector being used in the ASYNC test

### PR DESCRIPTION
**Pull request by brtnfld**
*added h5pget_vol_cap_flags_f with test.
*added Fortran VOL capability definitions.
*print the VOL connector being used in the test.

### CODE REVIEW

**Review by jhendersonHDF — APPROVED** [review]

**Review by byrnHDF — APPROVED** [review]

File: fortran/testpar/async.F90 [hunk 1]
```diff
@@ -1313,12 +1314,15 @@ PROGRAM async_test
...
         async_enabled = .FALSE.
+        IF(mpi_rank==0) WRITE(*,'(A,/)') "NATIVE"
      ELSE
+        IF(mpi_rank==0) WRITE(*,'(A,/)') "DAOS"
         CALL H5Vlregister_connector_by_name_f("daos", vol_id, hdferror)
         CALL check("H5Vlregister_connector_by_name_f", hdferror, total_error)
      ENDIF
 
   ELSE
+     IF(mpi_rank==0) WRITE(*,'(A,/)') "ASYNC"
```
Comment (derobins) [inline fortran/testpar/async.F90]: What if the connector under test isn't one of these connectors?

Comment (brtnfld) [reply fortran/testpar/async.F90]: Then it prints Native

Comment (brtnfld) [reply fortran/testpar/async.F90]: Will add flags API to get this information.

Comment (lrknox) [reply fortran/testpar/async.F90]: In this PR or in another one?

Comment (brtnfld) [reply fortran/testpar/async.F90]: This PR.

File: release_docs/RELEASE.txt [hunk 2]
```diff
@@ -499,7 +499,8 @@ Bug Fixes since HDF5-1.14.0 release
 
     Fortran API
     -----------
-    -
+    - Added h5pget_vol_cap_flags_f and related  Fortran VOL
```
Comment (hyoklee) [inline release_docs/RELEASE.txt]: Why 2 whitespace between **related** and **Fortran**?

**Review by hyoklee — CHANGES_REQUESTED** [review]

**Review by hyoklee — APPROVED** [review]

**Review by derobins — APPROVED** [review]

**Pull request merged**